l10n_ar_stock/models/res_company.py

```diff
@@ -38,7 +38,7 @@ def arba_cot_connect(self):
         cuit = self.partner_id.ensure_vat()
 
         if not self.arba_cot:
-            raise UserError(_(
```

Comment:
please could you improve this trasnlation, from new odoo versions you can pass the arguments of the string to be transatle inside the method _(). Instead of `_('string to translate %s') % arg` you could simplify code with `_('string to translate %s', arg)`. Can you do apply this in the translations you find?

Also, we can use max 120 characters, so if you want/can it to leave it in on line will be also good. 

Example: In the next example you see that we were using 3 lines, with this advice we have only on line. 

```python
 raise UserError(self.env._('You must configure ARBA COT on company %s', self.name)
```